justfile

```diff
@@ -207,6 +207,6 @@ clone-submodule dir url sha:
   cd {{dir}} ; git fetch --depth=1 origin {{sha}} ; git reset --hard {{sha}}
 
 website path:
-  cargo run -p website -- linter-rules --table {{path}}/src/docs/guide/usage/linter/generated-rules.md --rule-docs {{path}}/src/docs/guide/usage/linter/rules
+  cargo run -p website -- linter-rules --table {{path}}/src/docs/guide/usage/linter/generated-rules.md --rule-docs {{path}}/src/docs/guide/usage/linter/rules --git-ref $(git rev-parse HEAD)
```

Comment:
This uses a commit hash, but tags and branches are also supported here.  Might be worth using a tag for the website releases, but default to a commit hash for development.